src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -197,6 +197,16 @@ class DefaultStorageKeyAllocator
         return StorageKeyName::Formatted("g/su/%x", static_cast<unsigned>(index));
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
+    // Scene Storage
+    static StorageKeyName SceneFabricList()
+    {
+        return StorageKeyName::FromConst("g/gfl");
+    } // shares key with group fabric list to minimize flash usage
```

Comment:
This implementation detail means that the SceneTableImpl must ALWAYS be used along with the default GroupDataProviderImpl. You need to give details about these assumptions *and verify the configuration selection at build time*.

Reply:
My understanding is that if someone would implement a different group data provider, they would have to use a different key than this one, thus the Scene Table would function the same. Is this not the case? That being said, this StorageKeyName was removed and GroupFabricList is now used in SceneTableImpl.cpp.

Reply:
Please confirm regarding this point.